pkg/shared/types.go

```diff
@@ -36,6 +36,11 @@ const (
 	// DefaultNamingScheme - default naming scheme if not specified in
 	// the configCR
 	DefaultNamingScheme = "UID"
+
+	// DefaultMasterEncryptionKey - default master encryption key
+	// if not specified in configCR
+	// Generated 32 byte key for AES-256
+	DefaultMasterEncryptionKey = "7fc051adc074d25134d1987b978046fb77334c889e1997de3136854884241b9f"
```

Comment:
Currently it's supposed to conform AES key format. We can either add validation for it in OpenAPI (CRD) or allow to use any passphrase and transform it into encryption key by some hashing algorithm.

Reply:
IMO we shouldn't have any hardcoded default for the master key. If KD sees that there is no master key in its config, it should generate a random one and store it in the config.

Reply:
Oh, it makes sense actually. It's much safer